eispy/calibration/data_calibration.py

```diff
@@ -136,12 +136,15 @@ def _get_pixel_solid_angle(detector, slit):
     omega-d in the EIS software note 02.
     """
     mirror_focus_length = 1938.68 * u.mm
+    print(slit)
     if slit == 1:
```

Comment:
yeah, the slit number was coming out as zero, so this was a hack, no idea what the purpose of this function is  though.